From 2623cf77fa071106036450f4be9bd0c92451d858 Mon Sep 17 00:00:00 2001
From: Yohann Leon <yohann@leon.re>
Date: Fri, 9 Oct 2020 21:17:44 +0200
Subject: [PATCH] fix: spotify widget wrong size

---
 app/embeds/Spotify.js | 14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

diff --git a/app/embeds/Spotify.js b/app/embeds/Spotify.js
index 31728309..bb0fc49e 100644
--- a/app/embeds/Spotify.js
+++ b/app/embeds/Spotify.js
@@ -25,11 +25,21 @@ export default class Spotify extends React.Component<Props> {
   render() {
     const normalizedPath = this.pathname.replace(/^\/embed/, "/");
 
+    var height;
+
+    if (normalizedPath.includes("episode") || normalizedPath.includes("show")) {
+      height = 232;
+    } else if (normalizedPath.includes("track")) {
+      height = 80;
+    } else {
+      height = 380;
+    }
+
     return (
       <Frame
         {...this.props}
-        width="300px"
-        height="380px"
+        width="100%"
+        height={`${height}px`}
         src={`https://open.spotify.com/embed${normalizedPath}`}
         title="Spotify Embed"
         allow="encrypted-media"
-- 
2.28.0

