From f45eb06b0058b865db78028fb7f04b2943222527 Mon Sep 17 00:00:00 2001
From: Yohann Leon <yohann@leon.re>
Date: Sun, 27 Feb 2022 15:26:41 +0100
Subject: [PATCH] fix: apply teamlogo patch

---
 app/components/TeamLogo.ts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/app/components/TeamLogo.ts b/app/components/TeamLogo.ts
index 2eb5d144..af0f08b1 100644
--- a/app/components/TeamLogo.ts
+++ b/app/components/TeamLogo.ts
@@ -7,7 +7,6 @@ const TeamLogo = styled.img<{ width?: number; height?: number; size?: string }>`
     props.height ? `${props.height}px` : props.size || "38px"};
   border-radius: 4px;
   background: ${(props) => props.theme.background};
-  border: 1px solid ${(props) => props.theme.divider};
   overflow: hidden;
   flex-shrink: 0;
 `;
-- 
2.20.1

